scripts/bench.py

```diff
@@ -0,0 +1,55 @@
+#!/usr/bin/python
+import argparse
+import os
+import sys
+
+from monotonic import monotonic
+
+from detect_secrets.core.usage import PluginOptions
+
+if os.name == 'posix' and sys.version_info[0] < 3:
+    import subprocess32 as subprocess
+else:
+    import subprocess
+
+
+def getArguments():
+    parser = argparse.ArgumentParser(description='Run some benchmarks.')
+    parser.add_argument(
+        'rest_args',
+        nargs=argparse.REMAINDER,
+        help='filenames to check or detect-secrets compatible arguments',
+    )
+    args = parser.parse_args()
+    return ['../.'] if len(args.rest_args) == 0 else args.rest_args
+
+
+def runBench(flags, arguments):
+    startTime = monotonic()
+    subprocess.call('detect-secrets scan'.split() + arguments + flags, stdout=subprocess.DEVNULL)
+    return monotonic() - startTime
+
+
+def printLine(name, time):
+    print('{:<20s}{:>20s}s'.format(name, str(time)))
+
+
+flags = list()
+aditionalArguments = getArguments()
+
+# Fill flags list with PluginOptions flag text
+for c, value in enumerate(PluginOptions.all_plugins, 0):
+    flags.append(value.disable_flag_text)
+
+# Run bechmarks for all the cases
+for c, value in enumerate(flags, 0):
+    temp = list(flags)
+    temp.pop(c)
```

Comment:
I don't think this will do what you think it will do.

```
$ cat sample.py
a = [1, 2, 3]
b = a
b.pop(0)

print(a)
print(b)
$ python sample.py
[2, 3]
[2, 3]
```

For a really good talk of why this is, check out https://www.youtube.com/watch?v=_AEJHKGk9ns

Reply:
Um. I thought that... If I want to exclude all plugins except one in each run this was the expected result:
```
➜  ~ cat something.py 
flag_list = ['1','2','3','4']
for flag_number, flag in enumerate(flag_list):
    temp = list(flag_list)
    temp.pop(flag_number)
    print(temp)
➜  ~ python something.py 
['2', '3', '4']
['1', '3', '4']
['1', '2', '4']
['1', '2', '3']
```
Maybe there are other cleaner ways to do this! What do you think? I will take a look at the talk lately today, thank you :blush:

Reply:
Oh! Never mind -- you're absolutely right!

I guess the `list` was sufficient to copy the data, rather than type-casting. dgzlopes++

Reply:
Closing this one as resolved then :)